forms-web-app/src/views/appellant-submission/appeal-statement.njk

```diff
@@ -105,10 +105,10 @@
           text: "File size should be no more than " + fileSizeLimits.appealStatementMaxFileSize | formatBytes
         }) }}
 
-        {% if appeal['appeal-upload'] %}
+        {% if appeal['yourAppealSection']['appealStatement']['uploadedFile'].name %}
```

Comment:
minor - I think if using camel case we could use `appeal.yourAppealSection.appealStatement.uploadedFile.name` - only reason for bracket syntax was cus of the kebab case.

No need to change though

Reply:
You are right, I'll change it in my next PR